### ISSUE TITLE
`Exam mode`: pressing "override assessment" on an exercise, the button is still enabled. Even if there are not new changes

### ISSUE THREAD

**Issue by Santia-go** [ls1intum/Artemis#8932]
### Describe the bug

When I solve a complaint and involve a new assessment after pressing "override assessment" on an exercise, the button is still enabled. Even if there are not new changes

### To Reproduce

1. Solve a complaint and change the grade (add new feedback)
2. Check that the override assessment is enable 
3. press it and verify the grade was saved
4. Verify the button is still enabled even if you didn't add new feedback 

### Expected behavior

The button should not be enabled if no changes are there. 

### Which version of Artemis are you seeing the problem on?

7.3.0

### What browsers are you seeing the problem on?

Chrome

**Comment by florian-glombik** [comment]
I agree that this is a better approach, I changed the label as I do not think that this is a bug but rather an additional functionality
